libs/tup-components/src/projects/ProjectsListing/ProjectsNavbar.tsx

```diff
@@ -22,6 +22,7 @@ export const ProjectsNavbar: React.FC = () => {
       >
         <div>Inactive Projects</div>
       </QueryNavItem>
+      <div className={styles['project-navbar-spacer']} />
```

Comment:
Thank you for faithfully matching the design. Please remove this div and its styles.

I know you mimic the line beneath the links, but that is one of those things a lead designer will agree with me is not needed (nor desired)[^1], but no designer follows-up to remove the line from the design. I will ask them to remove this line.

[^1]: Another example is lines beneath tables (as on [portal design](https://xd.adobe.com/view/cc3dc245-2792-4e10-bdc1-50d2a7d32979-996e/screen/2395f5ea-b96e-4298-a443-ea3b596031ac/specs/) but not in [component design](https://xd.adobe.com/view/b37b291a-7798-4814-90f3-7ac63c48a3e0-e628/screen/907a5f5c-5b46-441f-95f6-f0e50f529c65/specs/))... except that I was later asked to add them to CMS 😵‍💫.

Reply:
I just [asked design (privately)](https://tacc-team.slack.com/archives/G01K9ET7JM7/p1676919720526539) to confirm this.

Reply:
<details><summary>Verified.</summary>

![remove border under inactive projects](https://user-images.githubusercontent.com/62723358/220184066-1dd05f3a-ff2d-4275-aec4-bcd80d2e5614.png)

</details>